boards/riscv/esp32c3_devkitm/doc/index.rst

```diff
@@ -51,21 +51,19 @@ On Linux and macOS:
 .. code-block:: console
 
    export ZEPHYR_TOOLCHAIN_VARIANT="espressif"
-   export ESPRESSIF_TOOLCHAIN_PATH="${HOME}/.espressif/tools/riscv32-esp-elf/1.24.0.123_64eb9ff-8.4.0/riscv32-esp-elf"
+   export ESPRESSIF_TOOLCHAIN_PATH="${HOME}/.espressif/tools/zephyr"
```

Comment:
just curious.

It seems the espressif toolchain is now expected to be installed under a different folder per default.
It that part of the espressif installer itself, or is there some missing doc update on toolchain installation ?

Also what about users having already having the espressif toolchain installed ?
How will they be aware that they must now move their current
`.espressif/tools/riscv32-esp-elf` into `.espressif/tools/zephyr/riscv32-esp-elf` instead.

Also, is `1.24.0.123_64eb9ff-8.4.0` no longer part of the toolchain installation path ?

Reply:
@tejlmand I have explained that already in this PR. Can you check previous comments? The main reason is having the same export PATH for Espressif toolchain to all SoCs, not depending on folder names and versioning as it is right now.

So, as you can see in the document update in this PR, no more `1.24.0.123_64eb9ff-8.4.0` or toolchain name explicit in folder path.  

`west espressif install` is the download agent responsible to download and extract the toolchain in proper place (or user can manually download it as well).  

> Also what about users already having the espressif toolchain installed ?
When building, it will give them cmake error with a message to re-configure toolchain path. 
Is there any recommendation on how to inform users about this change besides changing the ESP32 documentation?

Reply:
This is the output when running  `west espressif install`. It also informs the user how to export path into the environment.
```
sylvio@pc:~/projects/espressif/zephyr/zephyr$ west espressif install
=== downloading ESP-IDF tools..
Installing tools: xtensa-esp32-elf, xtensa-esp32s2-elf, riscv32-esp-elf, openocd-esp32
Skipping xtensa-esp32-elf@esp-2020r3-8.4.0 (already installed)
Skipping xtensa-esp32s2-elf@esp-2020r3-8.4.0 (already installed)
Skipping riscv32-esp-elf@1.24.0.123_64eb9ff-8.4.0 (already installed)
Skipping openocd-esp32@v0.10.0-esp32-20200709 (already installed)
=== downloading ESP-IDF tools completed
The toolchain has been downloaded to /home/sylvio/.espressif/tools/zephyr
Export or set the following variables into the environment:
export ESPRESSIF_TOOLCHAIN_PATH="/home/sylvio/.espressif/tools/zephyr"
export ZEPHYR_TOOLCHAIN_VARIANT="espressif"
```

Reply:
> @tejlmand I have explained that already in this PR.

If you refer to this comment https://github.com/zephyrproject-rtos/zephyr/pull/38540#issuecomment-922276759, then you only describe why, but my question was about who / how that path was ensured.
The installer or a guide ?

Now that question you answer with the `west espressif install` sentence. 

> > Also what about users already having the espressif toolchain installed ?
>
> When building, it will give them cmake error with a message to re-configure toolchain path.

No, you give them an error message requesting users to re-install something that is already installed.
I think we should ensure that if users have a setup with the old style then that will still be supported.

That will allow users to keep working and update the toolchain when a new revision is released and they update anyway.

By the way, your `west espressif install` fails on my machine.
```
west espressif install
=== downloading ESP-IDF tools..
/usr/bin/env: ‘python’: No such file or directory
FATAL ERROR: command exited with status 127: ./tools/idf_tools.py --tools-json=tools/zephyr_tools.json install
```
because I only have `python3` and no `python` link.
Why not use the python that was already invoked with `west` ?